pre-docs/control_repo.md

```diff
@@ -0,0 +1,21 @@
+# The control repository pattern
+
+The control repository pattern is useful for creating a project that contains your tasks and plans - along with declaring modules they depend on - for others to run. It's based on the idea originally [developed for Puppet](https://github.com/puppetlabs/best-practices/blob/master/control-repo-contents.md) of a repository that acts as a single location for an organization's configuration. The base example can be found [here](https://github.com/puppetlabs/control-repo).
```

Comment:
```suggestion
The control repository pattern is useful for creating a project that contains your tasks and plans - along with declaring modules they depend on - for others to run. It's based on the idea originally [developed for Puppet](https://github.com/puppetlabs/best-practices/blob/master/control-repo-contents.md) of a repository that acts as a single location for an organization's configuration. A basic example of a control repository with Bolt content can be found [here](https://github.com/puppetlabs/control-repo).
```

Reply:
That's not what's there though. The link was to an existing repo with an example Puppet control repo.

Reply:
Oh jeez, sorry, I thought this linked to the azure repo.